blocks/into/__title/into__title.css

```diff
@@ -0,0 +1,37 @@
+.into__title{
+    display: flex;
+    width: 300px;
+    height: 174px;
```

Comment:
Why would you need height here? Try adding padding top and bottom, it should do the trick.